<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmationWindow</class>
 <widget class="QWidget" name="ConfirmationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Part Confirmation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
   /* --- Unified Color Scheme --- */
   #ConfirmationWindow {
       background-color: #f0f2f5;
       font-family: Arial, sans-serif;
   }

   QLabel {
       color: #333;
       font-size: 22pt;
       font-weight: bold;
   }

   QPushButton {
       background-color: #1976d2;
       color: white;
       font-size: 14pt;
       font-weight: bold;
       padding: 10px 20px;
       border-radius: 8px;
       border: none;
       min-width: 180px;
       min-height: 50px;
   }

   QPushButton:hover {
       background-color: #1565c0;
   }
   </string>
  </property>

  <layout class="QVBoxLayout" name="mainLayout" stretch="2,3,1">
   <!-- Title Label -->
   <item>
    <widget class="QLabel" name="confirmationLabel">
     <property name="text">
      <string>Confirm Your Selection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>

   <!-- Centered Selected Part Box -->
   <item alignment="Qt::AlignCenter">
    <widget class="QToolButton" name="selectedPartButton">
     <property name="text">
      <string>Part X</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QToolButton {
           background-color: white;
           color: black;
           border: 4px solid #1976d2;  /* Blue border */
           border-radius: 10px;
           font-size: 16pt;
           font-weight: bold;
           min-width: 250px;
           min-height: 150px;
       }

       QToolButton:hover {
           background-color: #f5f5f5;
       }
      </string>
     </property>
    </widget>
   </item>

   <!-- Navigation Buttons -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
